src/crypto/rand/rand_stm32.go

```diff
@@ -0,0 +1,40 @@
+//go:build stm32wle5
+// +build stm32wle5
```

Comment:
Can we remove this duplicate (old syntax)?

Reply:
Not yet, because we still support Go 1.15.